src/buyingcatalogue/Chart.yaml

```diff
@@ -102,6 +102,10 @@ dependencies:
   condition: admin-ac-tests.enabled
   version: 1.1.1
   repository: "@gpitfuturesdevacr"
+- name: of-ac-tests
+  condition: of-ac-tests.enabled
+  version: 0.4.0-alpha
```

Comment:
As discussed, I believe this version needs to be 0.4.0-alpha.25 until it is deployed to master